### ISSUE TITLE
Hold & resume threads

### ISSUE THREAD

**Issue by StephenDaDev** [kyb3r/modmail#340]
Like when you need to do check something, you can put it on hold. It will send a message, and change name of channel name to [HOLD] USERNAME#XXXX, and block all messages until it's resume also pause time for auto close. Resume will say remove [HOLD] in name, allow messages, and resume timer for messages. Example: Command for hold ?hold. Command for resume ?resume

*Suggestion by Discord User ShamrockBot#1235**

**Comment by allipsee** [comment]
I think this will be a cool feature for ModMail
(Btw Hi Discord )

**Comment by nickkuro** [comment]
I agree.

**Comment by Weirdllamas** [comment]
Clarification, would the hold command take in a time argument for it to auto-resume at? Or would this be just be to indefinitely hold it until somebody did ?resume

**Comment by StephenDaDev** [comment]
From what it sounds like it would be indefinite until resumed or closed.

**Comment by Weirdllamas** [comment]
Oh wait, isn't this the same as ?block and ?unblock ?

**Comment by StephenDaDev** [comment]
+ pause timed close

**Comment by Weirdllamas** [comment]
okay i mean i guess i'll look at it. I got time to kill

**Comment by StephenDaDev** [comment]
This would also disallow messages to be sent from either side while enabled
